fastlane/Fastfile

```diff
@@ -16,13 +16,20 @@ private_lane :merged_prs_since_last_release do |options|
   github_label = options[:github_label]
   earliest_datetime = options[:earliest_datetime]
   ensure_git_log_includes_pr = options[:ensure_git_log_includes_pr]
+  
+  base_branch_name = if repo == "android-news-app"
+    "main"
```

Comment:
Maybe we could pass the branch name in with the options? That way we won't need changes to this file if other repos start using this action (or if/when iOS changes name).

Reply:
Thanks @SLambrakis. That's a much better approach! We can provide branch name in options from Android, and use `master` if there isn't any provided.